CONTRIBUTING.md

```diff
@@ -124,7 +124,8 @@ in order to craft an excellent pull request:
 5. Make sure all the tests are still passing.
 
    ```bash
-   fake build
+   cd tests/Saturn.UnitTests/
```

Comment:
does the command `dotnet fake build -t Test` work here instead? This is the command used in the [CI build script](https://github.com/SaturnFramework/Saturn/blob/master/.github/workflows/build.yml#L23) and would help ensure these two results are as similar as possible.

Similarly, the instructions below should use this command as the preferred way to run the tests IMO.  Moving to a specific test can then delve into the `dotnet run` workflow, or even delegate to Expecto's documentation.

Reply:
I'll test it here, good catch

Reply:
`dotnet fake build -t Test` ran from the root level, so I'll change the CONTRIBUTING.md to call it, since it is easier, and like you mentioned this is the command used in the CI build script

> Similarly, the instructions below should use this command as the preferred way to run the tests IMO.

Alright, I'll add this as the preferred way of running the tests but I still would like to keep the other way since it would be easier for newcomers to know how to run their specific test scenarios.